ApiBundle/Transformer/TransformerWithGroupContextInterface.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace OpenOrchestra\ApiBundle\Transformer;
+
+use OpenOrchestra\BackofficeBundle\Model\GroupInterface;
+use OpenOrchestra\BaseApi\Facade\FacadeInterface;
+use OpenOrchestra\BaseApi\Transformer\TransformerInterface;
+
+/**
+ * Interface TransformerWithGroupContextInterface
+ */
+interface TransformerWithGroupContextInterface extends TransformerInterface
```

Comment:
I would also change the part `GroupContext` as it is already used for a different part of the application